autospec/qt-modules.txt

```diff
@@ -0,0 +1,61 @@
+3danimation Qt53DAnimation
```

Comment:
Can you please change the format of this file to
```
# comment explaining the purpose and format of this file
key, value
key, value
...
```
to be consistent with the other configuration files in autospec (`failed_commands`, `maven_jars`). Then the file can be read in using `config.read_pattern_conf()`.

Reply:
You can also drop the `.txt` extension as it isn't needed.

Reply:
Will do.